python/feathub/feature_tables/sinks/kafka_sink.py

```diff
@@ -38,17 +40,26 @@ def __init__(
                              etc.
         :param producer_properties: Optional. If it is not None, it contains the extra
                                     kafka producer properties.
+        :param key_format_config: Optional. The configuration of the format for Kafka
```

Comment:
Would it be better to name it `key_format_properties` for consistency with `producer_properties`?

We can simplify `properties` to `props` if you prefer.